xbmc/cores/AudioEngine/Interfaces/AE.h

```diff
@@ -76,168 +85,239 @@ class IAE
   IAE() = default;
   virtual ~IAE() = default;
 
-  /**
-   * Initializes the AudioEngine, called by CFactory when it is time to initialize the audio engine.
+  /*!
+   * \brief Initializes the AudioEngine, called by CFactory when it is time to initialize the audio engine.
+   *
    * Do not call this directly, CApplication will call this when it is ready
    */
   virtual void Start() = 0;
 public:
-  /**
-   * Called when the application needs to terminate the engine
+  using Stream = std::unique_ptr<IAEStream, IAEStreamDeleter>;
+  using Sound = std::unique_ptr<IAESound, IAESoundDeleter>;
```

Comment:
may i suggest to call these `StreamPtr` and `SoundPtr`? we're using `AddonPtr`, `FileItemPtr` etc. already across the codebase. i was just about to propose removing the aliases, but now i think it's better to keep @garbear relaxed after i saw his comment https://github.com/xbmc/xbmc/pull/20292#discussion_r725721242 :)

Reply:
Done :slightly_smiling_face:. I wasn't sure if having `Ptr` as suffix for both `std::unique_ptr` and `std::shared_ptr` types wouldn't be confusing but it's probably OK.

Reply:
thx, i will give it another round of testing. if nothing pops up i'll merge tomorrow